pkgs/top-level/qt5-packages.nix

```diff
@@ -174,7 +174,7 @@ makeScopeWithSplicing' {
 
         kreport = callPackage ../development/libraries/kreport { };
 
-        kquickimageedit = callPackage ../development/libraries/kquickimageedit { };
+        kquickimageedit = callPackage ../development/libraries/kquickimageedit/0.3.0.nix { };
```

Comment:
How do we reference 0.5.1?

```suggestion
        kquickimageedit_0_3_0 = callPackage ../development/libraries/kquickimageedit/0.3.0.nix { };
        kquickimageedit = callPackage ../development/libraries/kquickimageedit { };
```

Reply:
0.3.0 is really the last version that supports Qt 5, so `libsForQt5.kquickimageedit` can't be updated as is.

Reply:
I see.

Note: libsForQt5.kquickimageedit is 0.3.0 and kdePackages.kquickimageedit is 0.5.1.